### PR TITLE
RDS: updates Instance, Cluster, and ClusterInstance to use `WithoutTimeout` CRUDI functions

### PR DESCRIPTION
**Pull request by gdavison**
updates `aws_db_instance`, `aws_rds_cluster`, and `aws_rds_cluster_instance` to use `WithoutTimeout` CRUD functions and `Context`-aware Import functions.

Also ensures that `Context`-aware functions are used in other resources when `WithoutTimeout` CRUD functions are used.

### Output from Acceptance Testing

Acceptance test failures are pre-existing

```
$ make testacc PKG=rds TESTS='TestAccRDSInstance|TestAccRDSSnapshotCopy|TestAccRDSReservedInstance|TestAccRDSCluster' ACCTEST_TIMEOUT=5h

--- SKIP: TestAccRDSCluster_SnapshotIdentifierEngineMode_serverless (0.00s)
--- PASS: TestAccRDSCluster_iamAuth (129.02s)
--- PASS: TestAccRDSCluster_backupsUpdate (225.55s)
--- FAIL: TestAccRDSCluster_ReplicationSourceIdentifier_kmsKeyID (1.63s)
--- PASS: TestAccRDSCluster_engineVersionWithPrimaryInstance (1185.56s)
--- PASS: TestAccRDSCluster_encrypted (152.97s)
--- PASS: TestAccRDSCluster_copyTagsToSnapshot (217.46s)
--- PASS: TestAccRDSCluster_kmsKey (147.45s)
--- PASS: TestAccRDSCluster_networkType (209.22s)
--- PASS: TestAccRDSCluster_updateIAMRoles (204.51s)
--- PASS: TestAccRDSCluster_identifierPrefix (158.43s)
--- PASS: TestAccRDSCluster_identifierGenerated (168.60s)
--- PASS: TestAccRDSCluster_tags (192.43s)
--- PASS: TestAccRDSCluster_disappears (141.37s)
--- PASS: TestAccRDSCluster_basic (169.06s)
--- FAIL: TestAccRDSCluster_EnabledCloudWatchLogsExports_postgresql (14.36s)
--- FAIL: TestAccRDSCluster_iops (7.62s)
--- FAIL: TestAccRDSCluster_allocatedStorage (8.18s)
--- PASS: TestAccRDSCluster_EnabledCloudWatchLogsExports_mySQL (245.97s)
--- FAIL: TestAccRDSCluster_dbClusterInstanceClass (7.79s)
--- PASS: TestAccRDSCluster_missingUserNameCausesError (6.65s)
--- PASS: TestAccRDSCluster_engineVersion (478.73s)
--- PASS: TestAccRDSCluster_engineMode (412.10s)
--- FAIL: TestAccRDSCluster_EngineMode_parallelQuery (9.56s)
--- PASS: TestAccRDSCluster_EngineMode_multiMaster (145.60s)
--- PASS: TestAccRDSCluster_EngineMode_global (201.12s)
--- PASS: TestAccRDSCluster_GlobalClusterIdentifierEngineMode_global (143.41s)
--- PASS: TestAccRDSCluster_deletionProtection (226.57s)
--- PASS: TestAccRDSCluster_takeFinalSnapshot (224.43s)
--- FAIL: TestAccRDSCluster_storageType (8.40s)
--- PASS: TestAccRDSCluster_SnapshotIdentifierEngineVersion_different (471.32s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_vpcSecurityGroupIDs (456.42s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_tags (433.32s)
--- PASS: TestAccRDSCluster_allowMajorVersionUpgradeWithCustomParametersApplyImm (2578.33s)
--- PASS: TestAccRDSCluster_PointInTimeRestore_enabledCloudWatchLogsExports (987.45s)
--- PASS: TestAccRDSCluster_dbSubnetGroupName (875.26s)
--- PASS: TestAccRDSCluster_pointInTimeRestore (1075.01s)
--- PASS: TestAccRDSCluster_availabilityZones (1062.05s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_encryptedRestore (1257.32s)
--- PASS: TestAccRDSCluster_backtrackWindow (1571.75s)
--- PASS: TestAccRDSCluster_allowMajorVersionUpgrade (2492.82s)
--- PASS: TestAccRDSCluster_enableHTTPEndpoint (1641.54s)
--- PASS: TestAccRDSCluster_onlyMajorVersion (1884.58s)
--- PASS: TestAccRDSCluster_SnapshotIdentifierVPCSecurityGroupIDs_tags (461.69s)
--- FAIL: TestAccRDSCluster_SnapshotIdentifierEngineMode_parallelQuery (6.94s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_preferredMaintenanceWindow (447.67s)
--- PASS: TestAccRDSCluster_port (256.75s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_masterPassword (398.72s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_kmsKeyID (429.19s)
--- PASS: TestAccRDSCluster_allowMajorVersionUpgradeWithCustomParameters (2293.19s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_deletionProtection (492.44s)
--- FAIL: TestAccRDSCluster_GlobalClusterIdentifier_replicationSourceIdentifier (1.83s)
--- FAIL: TestAccRDSCluster_GlobalClusterIdentifier_primarySecondaryClusters (1.82s)
--- PASS: TestAccRDSCluster_SnapshotIdentifierEngineMode_provisioned (407.17s)
--- PASS: TestAccRDSCluster_GlobalClusterIdentifierEngineMode_provisioned (142.71s)
--- FAIL: TestAccRDSCluster_GlobalClusterIdentifier_secondaryClustersWriteForwarding (1.98s)
--- PASS: TestAccRDSCluster_Scaling_defaultMinCapacity (296.56s)
--- FAIL: TestAccRDSCluster_GlobalClusterIdentifierEngineModeGlobal_add (165.96s)
--- PASS: TestAccRDSCluster_snapshotIdentifier (415.93s)
--- PASS: TestAccRDSCluster_serverlessV2ScalingConfiguration (245.49s)
--- PASS: TestAccRDSCluster_SnapshotIdentifierEngineVersion_equal (484.73s)
--- FAIL: TestAccRDSCluster_GlobalClusterIdentifierEngineModeGlobal_update (195.74s)
--- PASS: TestAccRDSCluster_scaling (363.93s)
--- PASS: TestAccRDSCluster_GlobalClusterIdentifierEngineModeGlobal_remove (239.32s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_masterUsername (436.56s)
--- PASS: TestAccRDSCluster_SnapshotIdentifier_preferredBackupWindow (436.36s)

--- PASS: TestAccRDSInstance_noDeleteAutomatedBackups (699.85s)
--- SKIP: TestAccRDSInstance_coIPEnabled (0.34s)
--- SKIP: TestAccRDSInstance_CoIPEnabled_snapshotIdentifier (0.36s)
--- SKIP: TestAccRDSInstance_CoIPEnabled_restoreToPointInTime (0.33s)
--- SKIP: TestAccRDSInstance_CoIPEnabled_enabledToDisabled (0.12s)
--- SKIP: TestAccRDSInstance_CoIPEnabled_disabledToEnabled (0.11s)
--- SKIP: TestAccRDSInstance_ReplicateSourceDB_deletionProtection (0.00s)
--- SKIP: TestAccRDSInstance_S3Import_basic (0.00s)
--- SKIP: TestAccRDSInstance_SnapshotIdentifier_dbSubnetGroupNameRAMShared (0.70s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_nameGenerated (983.06s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_deletionProtection (1042.03s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_AssociationRemoved (1107.05s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_availabilityZone (1161.59s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_dbSubnetGroupNameVPCSecurityGroupIDs (1190.67s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_maintenanceWindow (1196.36s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_backupWindow (1199.30s)
--- SKIP: TestAccRDSInstance_SnapshotIdentifier_tagsRemove (0.00s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_autoMinorVersionUpgrade (1231.69s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_dbSubnetGroupName (1258.96s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_monitoring (1302.45s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_maxAllocatedStorage (1316.86s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_iamDatabaseAuthenticationEnabled (1318.35s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_namePrefix (1383.91s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_allocatedStorage (1390.71s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_io1Storage (1472.73s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_backupRetentionPeriodOverride (1580.60s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_backupRetentionPeriodUnset (1742.51s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_basic (1261.18s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_allowMajorVersionUpgrade (2026.28s)
--- PASS: TestAccRDSInstance_caCertificateIdentifier (370.95s)
--- PASS: TestAccRDSInstance_license (1042.81s)
--- PASS: TestAccRDSInstance_NoNationalCharacterSet_oracle (838.62s)
--- PASS: TestAccRDSInstance_NationalCharacterSet_oracle (893.71s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_caCertificateIdentifier (1427.20s)
--- PASS: TestAccRDSInstance_performanceInsightsRetentionPeriod (965.75s)
--- PASS: TestAccRDSInstance_performanceInsightsKMSKeyID (966.66s)
--- PASS: TestAccRDSInstance_PerformanceInsightsEnabled_enabledToDisabled (780.51s)
--- PASS: TestAccRDSInstance_EnabledCloudWatchLogsExports_postgresql (516.67s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_vpcSecurityGroupIDs (1583.86s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_port (1583.02s)
--- PASS: TestAccRDSInstance_PerformanceInsightsEnabled_disabledToEnabled (771.31s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_performanceInsightsEnabled (1292.39s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_parameterGroupNameSetOnReplica (1683.72s)
--- PASS: TestAccRDSInstance_separateIopsUpdate (705.29s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_parameterGroupNameReplicaCopiesValue (1648.70s)
--- PASS: TestAccRDSInstance_nameDeprecated (539.90s)
--- PASS: TestAccRDSInstance_RestoreToPointInTime_sourceIdentifier (1463.92s)
--- PASS: TestAccRDSInstance_RestoreToPointInTime_monitoring (1584.73s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_performanceInsightsEnabled (1484.93s)
--- PASS: TestAccRDSInstance_RestoreToPointInTime_sourceResourceID (1569.36s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_replicaMode (2323.55s)
--- PASS: TestAccRDSInstance_tags (567.13s)
--- PASS: TestAccRDSInstance_identifierGenerated (471.10s)
--- PASS: TestAccRDSInstance_disappears (510.25s)
--- PASS: TestAccRDSInstance_EnabledCloudWatchLogsExports_oracle (886.40s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_dbSubnetGroupName (2403.08s)
--- SKIP: TestAccRDSInstance_DBSubnetGroupName_ramShared (0.00s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_addLater (1389.32s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_nameGenerated (1466.58s)
--- PASS: TestAccRDSInstance_password (391.98s)
--- PASS: TestAccRDSInstance_deletionProtection (620.65s)
--- PASS: TestAccRDSInstance_optionGroup (555.59s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_availabilityZone (1540.86s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_backupRetentionPeriod (1871.51s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_allocatedStorage (1366.25s)
--- PASS: TestAccRDSInstance_subnetGroup (758.92s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_namePrefix (1573.29s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_allowMajorVersionUpgrade (1393.59s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_basic (1362.82s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_autoMinorVersionUpgrade (1495.73s)
--- PASS: TestAccRDSInstance_dbSubnetGroupName (550.42s)
--- PASS: TestAccRDSInstance_DBSubnetGroupName_vpcSecurityGroupIDs (434.79s)
--- PASS: TestAccRDSInstance_kmsKey (377.59s)
--- PASS: TestAccRDSInstance_networkType (800.89s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_allocatedStorageAndIops (1602.43s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_iops (1437.96s)
--- PASS: TestAccRDSInstance_iamAuth (524.91s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_multiAZ (1766.06s)
--- PASS: TestAccRDSInstance_customIAMInstanceProfile (2729.73s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_multiAZSQLServer (5403.06s)
--- PASS: TestAccRDSInstance_onlyMajorVersion (545.71s)
--- FAIL: TestAccRDSInstance_ReplicateSourceDB_parameterGroupTwoStep (2707.80s)
--- PASS: TestAccRDSInstance_cloudWatchLogsExport (455.02s)
--- PASS: TestAccRDSInstance_EnabledCloudWatchLogsExports_msSQL (929.15s)
--- PASS: TestAccRDSInstance_minorVersion (500.13s)
--- PASS: TestAccRDSInstance_basic (512.28s)
--- PASS: TestAccRDSInstance_identifierPrefix (541.72s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_networkType (1143.61s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_parameterGroupNameSameSetOnBoth (1358.95s)
--- PASS: TestAccRDSInstance_EnabledCloudWatchLogsExports_mySQL (973.55s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_parameterGroupNameDifferentSetOnBoth (1952.64s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_monitoring_sourceAlreadyExists (1515.81s)
--- PASS: TestAccRDSInstance_MonitoringRoleARN_removedToEnabled (737.48s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_iamDatabaseAuthenticationEnabled (1261.42s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_maintenanceWindow (1311.99s)
--- PASS: TestAccRDSInstance_MonitoringRoleARN_enabledToRemoved (828.37s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_multiAZ (1867.78s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_monitoring (1790.40s)
--- PASS: TestAccRDSInstance_monitoringInterval (1082.90s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_maxAllocatedStorage (1601.95s)
--- PASS: TestAccRDSInstance_MonitoringRoleARN_enabledToDisabled (947.70s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_vpcSecurityGroupIDs (1196.42s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_vpcSecurityGroupIDsTags (1105.10s)
--- PASS: TestAccRDSInstance_MySQL_snapshotRestoreWithEngineVersion (1656.76s)
--- PASS: TestAccRDSInstance_ReplicateSourceDBDBSubnetGroupName_vpcSecurityGroupIDs (2065.81s)
--- PASS: TestAccRDSInstance_portUpdate (641.99s)
--- SKIP: TestAccRDSInstance_ReplicateSourceDBDBSubnetGroupName_ramShared (0.07s)
--- PASS: TestAccRDSInstance_MSSQL_domainSnapshotRestore (3317.40s)
--- PASS: TestAccRDSInstance_MSSQL_tz (2063.73s)
--- PASS: TestAccRDSInstance_isAlreadyBeingDeleted (1018.09s)
--- PASS: TestAccRDSInstance_FinalSnapshotIdentifier_skipFinalSnapshot (883.75s)
--- PASS: TestAccRDSInstance_maxAllocatedStorage (2149.77s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_parameterGroupName (1524.07s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_port (1222.09s)
--- PASS: TestAccRDSInstance_SnapshotIdentifier_tags (1355.97s)
--- PASS: TestAccRDSInstance_allowMajorVersionUpgrade (534.86s)
--- PASS: TestAccRDSInstance_MSSQL_domain (4191.94s)
--- PASS: TestAccRDSInstance_ReplicateSourceDB_backupWindow (1436.24s)
--- PASS: TestAccRDSInstance_finalSnapshotIdentifier (923.08s)

--- SKIP: TestAccRDSReservedInstance_basic (0.00s)

--- PASS: TestAccRDSInstanceOffering_basic (21.10s)

--- PASS: TestAccRDSSnapshotCopy_disappears (713.71s)
--- PASS: TestAccRDSSnapshotCopy_tags (678.97s)
--- PASS: TestAccRDSSnapshotCopy_basic (706.30s)

--- PASS: TestAccRDSClusterActivityStream_basic (1802.09s)
--- PASS: TestAccRDSClusterActivityStream_disappears (1794.78s)

--- PASS: TestAccRDSClusterParameterGroup_GeneratedName_parameter (17.75s)
--- PASS: TestAccRDSClusterParameterGroup_disappears (12.99s)
--- PASS: TestAccRDSClusterParameterGroup_generatedName (22.18s)
--- PASS: TestAccRDSClusterParameterGroup_NamePrefix_parameter (26.50s)
--- PASS: TestAccRDSClusterParameterGroup_namePrefix (19.51s)
--- PASS: TestAccRDSClusterParameterGroup_withApplyMethod (24.14s)
--- PASS: TestAccRDSClusterParameterGroup_basic (88.08s)
--- PASS: TestAccRDSClusterParameterGroup_updateParameters (40.89s)
--- PASS: TestAccRDSClusterParameterGroup_caseParameters (52.51s)
--- PASS: TestAccRDSClusterParameterGroup_only (38.87s)

--- PASS: TestAccRDSInstanceRoleAssociation_basic (995.04s)
--- PASS: TestAccRDSInstanceRoleAssociation_disappears (816.00s)

--- PASS: TestAccRDSClusterSnapshotDataSource_mostRecent (303.37s)
--- PASS: TestAccRDSClusterSnapshotDataSource_dbClusterIdentifier (873.29s)
--- PASS: TestAccRDSClusterSnapshotDataSource_dbClusterSnapshotIdentifier (1087.92s)

--- PASS: TestAccRDSClusterSnapshot_basic (204.58s)
--- PASS: TestAccRDSClusterSnapshot_tags (257.30s)

--- PASS: TestAccRDSClusterInstance_PerformanceInsightsKMSKeyID_auroraPostgresql (816.68s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsKMSKeyIDAuroraPostgresql_defaultKeyToCustomKey (937.68s)
--- PASS: TestAccRDSClusterInstance_MonitoringRoleARN_enabledToDisabled (1221.65s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsKMSKeyIDAuroraMySQL2_defaultKeyToCustomKey (896.65s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsKMSKeyIDAuroraMySQL1_defaultKeyToCustomKey (893.91s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsEnabled_auroraMySQL2 (913.02s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsKMSKeyID_auroraMySQL1 (955.39s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsKMSKeyID_auroraMySQL2 (999.44s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsEnabled_auroraMySQL1 (924.59s)
--- PASS: TestAccRDSClusterInstance_PerformanceInsightsEnabled_auroraPostgresql (1017.37s)
--- PASS: TestAccRDSClusterInstance_performanceInsightsRetentionPeriod (1433.81s)
--- PASS: TestAccRDSClusterInstance_MonitoringRoleARN_removedToEnabled (1942.88s)
--- PASS: TestAccRDSClusterInstance_MonitoringRoleARN_enabledToRemoved (1695.29s)
--- FAIL: TestAccRDSClusterInstance_publiclyAccessible (123.48s)
--- PASS: TestAccRDSClusterInstance_az (1297.10s)
--- PASS: TestAccRDSClusterInstance_caCertificateIdentifier (1068.13s)
--- PASS: TestAccRDSClusterInstance_copyTagsToSnapshot (931.04s)
--- PASS: TestAccRDSClusterInstance_isAlreadyBeingDeleted (1058.44s)
--- PASS: TestAccRDSClusterInstance_disappears (935.33s)
--- PASS: TestAccRDSClusterInstance_monitoringInterval (1642.78s)
--- PASS: TestAccRDSClusterInstance_tags (1123.69s)
--- PASS: TestAccRDSClusterInstance_basic (928.59s)
--- PASS: TestAccRDSClusterInstance_kmsKey (1061.16s)
--- PASS: TestAccRDSClusterInstance_identifierPrefix (965.76s)
--- PASS: TestAccRDSClusterInstance_identifierGenerated (881.94s)

--- PASS: TestAccRDSInstanceDataSource_basic (1250.63s)

--- PASS: TestAccRDSClusterRoleAssociation_Disappears_role (574.90s)
--- PASS: TestAccRDSClusterRoleAssociation_Disappears_cluster (193.34s)
--- PASS: TestAccRDSClusterRoleAssociation_disappears (163.77s)
--- PASS: TestAccRDSClusterRoleAssociation_basic (187.80s)

--- PASS: TestAccRDSInstanceAutomatedBackupsReplication_kmsEncrypted (2141.71s)
--- PASS: TestAccRDSInstanceAutomatedBackupsReplication_retentionPeriod (2040.05s)
--- PASS: TestAccRDSInstanceAutomatedBackupsReplication_basic (2114.45s)

--- PASS: TestAccRDSClusterDataSource_basic (176.27s)

--- PASS: TestAccRDSClusterEndpoint_tags (1776.26s)
--- PASS: TestAccRDSClusterEndpoint_basic (1657.97s)
```

### CODE REVIEW

File: internal/service/rds/cluster_data_source.go [hunk 1]
```diff
@@ -136,11 +137,12 @@ func DataSourceCluster() *schema.Resource {
 }
 
 func dataSourceClusterRead(d *schema.ResourceData, meta interface{}) error {
+	ctx := context.TODO()
```
Comment (ewbankkit) [inline internal/service/rds/cluster_data_source.go:140]: Yould change `Read` to `ReadWithoutTimeout` for this data source and pass through the true `Context`.

File: internal/service/rds/instance_data_source.go [hunk 2]
```diff
@@ -184,10 +185,11 @@ func DataSourceInstance() *schema.Resource {
 }
 
 func dataSourceInstanceRead(d *schema.ResourceData, meta interface{}) error {
+	ctx := context.TODO()
```
Comment (ewbankkit) [inline internal/service/rds/instance_data_source.go:188]: Change `Read` to `ReadWIthoutTimeout` and pass through the real `Context`.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

```console
% make testacc TESTARGS='-run=TestAccRDSCluster_basic\|TestAccRDSInstance_basic\|TestAccRDSInstanceOffering_basic\|TestAccRDSSnapshotCopy_basic\|TestAccRDSClusterActivityStream_basic\|TestAccRDSClusterParameterGroup_basic\|TestAccRDSClusterSnapshot_basic\|TestAccRDSClusterInstance_basic\|TestAccRDSClusterEndpoint_basic\|TestAccRDSInstanceAutomatedBackupsReplication_basic' PKG=rds ACCTEST_PARALLELISM=2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/rds/... -v -count 1 -parallel 2  -run=TestAccRDSCluster_basic\|TestAccRDSInstance_basic\|TestAccRDSInstanceOffering_basic\|TestAccRDSSnapshotCopy_basic\|TestAccRDSClusterActivityStream_basic\|TestAccRDSClusterParameterGroup_basic\|TestAccRDSClusterSnapshot_basic\|TestAccRDSClusterInstance_basic\|TestAccRDSClusterEndpoint_basic\|TestAccRDSInstanceAutomatedBackupsReplication_basic -timeout 180m
=== RUN   TestAccRDSClusterActivityStream_basic
=== PAUSE TestAccRDSClusterActivityStream_basic
=== RUN   TestAccRDSClusterEndpoint_basic
=== PAUSE TestAccRDSClusterEndpoint_basic
=== RUN   TestAccRDSClusterInstance_basic
=== PAUSE TestAccRDSClusterInstance_basic
=== RUN   TestAccRDSClusterParameterGroup_basic
=== PAUSE TestAccRDSClusterParameterGroup_basic
=== RUN   TestAccRDSClusterSnapshot_basic
=== PAUSE TestAccRDSClusterSnapshot_basic
=== RUN   TestAccRDSCluster_basic
=== PAUSE TestAccRDSCluster_basic
=== RUN   TestAccRDSInstanceAutomatedBackupsReplication_basic
=== PAUSE TestAccRDSInstanceAutomatedBackupsReplication_basic
=== RUN   TestAccRDSInstance_basic
=== PAUSE TestAccRDSInstance_basic
=== RUN   TestAccRDSInstanceOffering_basic
=== PAUSE TestAccRDSInstanceOffering_basic
=== RUN   TestAccRDSSnapshotCopy_basic
=== PAUSE TestAccRDSSnapshotCopy_basic
=== CONT  TestAccRDSClusterActivityStream_basic
=== CONT  TestAccRDSCluster_basic
--- PASS: TestAccRDSCluster_basic (154.78s)
=== CONT  TestAccRDSInstanceOffering_basic
--- PASS: TestAccRDSInstanceOffering_basic (12.76s)
=== CONT  TestAccRDSSnapshotCopy_basic
--- PASS: TestAccRDSSnapshotCopy_basic (791.36s)
=== CONT  TestAccRDSClusterParameterGroup_basic
--- PASS: TestAccRDSClusterParameterGroup_basic (80.02s)
=== CONT  TestAccRDSClusterSnapshot_basic
--- PASS: TestAccRDSClusterSnapshot_basic (157.29s)
=== CONT  TestAccRDSInstance_basic
--- PASS: TestAccRDSInstance_basic (373.52s)
=== CONT  TestAccRDSInstanceAutomatedBackupsReplication_basic
--- PASS: TestAccRDSClusterActivityStream_basic (1871.25s)
=== CONT  TestAccRDSClusterInstance_basic
--- PASS: TestAccRDSClusterInstance_basic (1093.63s)
=== CONT  TestAccRDSClusterEndpoint_basic
--- PASS: TestAccRDSInstanceAutomatedBackupsReplication_basic (1400.75s)
--- PASS: TestAccRDSClusterEndpoint_basic (1774.64s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/rds	4744.194s
```

File: internal/service/rds/cluster_data_source.go (hunk 1 above)
Comment (gdavison) [reply internal/service/rds/cluster_data_source.go:140]: Good point

**Pull request merged**